site/layouts/partials/home/plugins.html

```diff
@@ -0,0 +1,65 @@
+<section class="pb-md-5 mb-5">
+  <div class="col-lg-8 mb-5">
+    <div class="masthead-followup-icon d-inline-block mb-3" style="--bg-rgb: var(--bs-warning-rgb);">
+      <svg class="bi fs-1"><use xlink:href="#plugin"></use></svg>
+    </div>
+    <h2 class="display-5 mb-3 fw-semibold lh-sm">Powerful JavaScript plugins without&nbsp;jQuery</h2>
+    <p class="lead fw-normal">
+      Add toggleable hidden elements, modals and offcanvas menus, popovers and tooltips, and so much more—all without jQuery. Bootstrap's JavaScript is HTML-first, meaning most plugins are added with <code>data</code> attributes in your HTML. Need more control? Include individual plugins programmatically.
+    </p>
+    <p class="d-flex justify-content-start lead fw-normal mb-md-0">
+      <a href="/docs/{{ .Site.Params.docs_version }}/getting-started/javascript/" class="icon-link fw-semibold">
+        Learn more about Bootstrap JavaScript
+        <svg class="bi"><use xlink:href="#arrow-right-short"></use></svg>
+      </a>
+    </p>
+  </div>
+  <div class="row gx-md-5">
+    <div class="col-lg-6 mb-3">
+      <h3 class="fw-semibold">Data attribute API</h3>
+      <p>Why write more JavaScript when you can write HTML? Nearly all of Bootstrap's JavaScript plugins feature a first-class data API, allowing you to use JavaScript just by adding <code>data</code> attributes.</p>
+      <div class="p-4 mb-3 border rounded-3">
+        <div class="dropdown">
+          <button class="btn btn-primary dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Dropdown
+          </button>
+          <ul class="dropdown-menu">
+            <li><button class="dropdown-item">Dropdown item</button></li>
+            <li><button class="dropdown-item">Dropdown item</button></li>
+            <li><button class="dropdown-item">Dropdown item</button></li>
+          </ul>
+        </div>
+      </div>
+
+      {{ highlight (printf `<div class="dropdown">
+<button class="btn btn-primary dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+  Dropdown
+</button>
+<ul class="dropdown-menu">
+  <li><button class="dropdown-item">Dropdown item</button></li>
```

Comment:
Nice, I prefer this version with buttons in this home page. However, they are missing `type="button"` (see https://getbootstrap.com/docs/5.2/components/dropdowns/#menu-items)